cmd/flags/root.go

```diff
@@ -65,6 +67,16 @@ func (r *Root) GetMaxDurationSeconds() int {
 	return r.MaxDurationSeconds
 }
 
+// MaxReadinessWaitSeconds returns the value of the max-readiness-wait-seconds parameter.
```

Comment:
I think this should match the name of the function.
I'm also wondering if we need the getters since the fields of the struct are capitalized and hence already exported.

Reply:
Fixed. I think the getter is to avoid exposing the entire root attributes